test/functional/harness/index.js

```diff
@@ -0,0 +1,50 @@
+var Environment   = require('./environment.js');
+var runMocha      = require('./run-mocha.js');
+var BrowserHelper = require('./browser-helper');
+
+exports.testCafe     = null;
+exports.browsersInfo = null;
+
+exports.run = function (ports, testPath, timeout, browserSettings, sauceLabsSettings) {

```

Comment:
we should add the `hostname` argument. We set it to `localhost` now, but we will change it for test runs on travis to os.hostname. So it'll be useful to create an opportunity to customize it